### DISCUSSION TITLE
onlyoffice_7.1以上的版本默认启用JWT，想问下ALIST什么时候支持？

### DISCUSSION THREAD

**Discussion by lxsexl** [AlistGo/docs#371] [Q&A]
ALIST安装好后的文档预览功能，包括微软和谷歌两个接口，微软要443端口才能正常预览（80，443这两个端口国内运营商是封锁的），谷歌都知道要梯子的。等于两个全都用不了。只有一个选择onlyoffice，不过新版本的默认开启JWT功能了。
我想知道现有版本支持JWT连接自建onlyoffice吗？如支持，该怎么操作？
如不支持，以后版本会添加吗？

**Comment by Pleasurecruise** [comment]
同问，不知道怎么配置iframe预览
